### ISSUE TITLE
Automated accessibility tool violations

### ISSUE THREAD

**Issue by hackartisan** [samvera-labs/clover-iiif#293]
We've added an accessibility scan step to our CI pipeline using [a11y-audit-elixir](https://github.com/angelikatyborska/a11y-audit-elixir), which wraps [axe-core](https://github.com/dequelabs/axe-core).

The following violations come from within clover.

The first is regarding the control icons (zoom / pan / rotate): 

      serious  [role="img"] elements must have an alternative text
     ┃ Learn more: https://dequeuniversity.com/rules/axe/4.10/role-img-alt?application=axeAPI
     ┃
     ┃ There is 1 node with this violation:
     ┃
     ┃ 1. <div id="openseadragon-cf44daeb-167f-46a3-86ae-560a7e56938c--1617087141" data-testid="clover-iiif-im...
     ┃
     ┃    Fix any of the following:
     ┃      - aria-label attribute does not exist or is empty
     ┃      - aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
     ┃      - Element has no title attribute
     ┃

The second is regarding the contents of the thumbnail row:

      minor  Alternative text of images should not be repeated as text
     ┃ Learn more: https://dequeuniversity.com/rules/axe/4.10/image-redundant-alt?application=axeAPI
     ┃
     ┃ There are 2 nodes with this violation:
     ┃
     ┃ 1. <img alt="1" data-testid="media-thumbnail-image" src="http://iiif.io/api/presentation/2.1/example/fi...
     ┃
     ┃    Fix all of the following:
     ┃      - Element contains <img> element with alt text that duplicates existing text
     ┃
     ┃ 2. <img alt="2" data-testid="media-thumbnail-image" src="http://iiif.io/api/presentation/2.1/example/fi...
     ┃
     ┃    Fix all of the following:
     ┃      - Element contains <img> element with alt text that duplicates existing text
     ┃

We are able to filter these out of our scan.

**Comment by manankharwar** [comment]
Since you're already using axe-core in CI, AccessOps could complement that with Chrome extension scanning for development and team dashboards to track the specific violations you're finding. https://accessops.dev/
